web/src/components/ProtocolBlockEditor.tsx

```diff
@@ -614,6 +614,12 @@ export function ProtocolBlockEditor(props: ProtocolBlockEditorProps) {
                         props.setBlock({ ...block, type: 'plate-sequencer', plateCount, plates });
                     }}
                 />
+                <ProtocolBlockNameEditor
+                    disabled={props.disabled}
+                    name={block.name}
+                    setName={name => props.setBlock({ ...block, type: 'start-timestamp', name })}
```

Comment:
I don't think this will pass typechecking right now, this should be:
```
name={block.timestampLabel}
setName={timestampLabel => props.setBlock({ ...block, type: 'plate-sequencer', timestampLabel })}
```

Reply:
Hm, so, the previous version did pass typechecking. But in the protocol editor it was basically showing the step name 2x... I had just copied over the code from the timestamp block blindly, not realizing I wouldn't need the "name" section. So, I just deleted this section.